fire/cli/søg/punkt.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+import click
+
+import fire.cli
+from fire.cli import firedb
+from . import søg
+
+
+@søg.command()
+@click.argument("ident")
+@click.option(
+    "-n",
+    "--antal",
+    default=20,
+    type=int,
+    help="Begræns antallet af fundne søgeresultater",
+)
+def punkt(ident: str, antal: int):
+    """
+    Søg efter et punkt ud fra dets ident
+
+    Søgeudtryk kan præciseres med wildcards givet ved %. Hvis ingen
+    wildcards angives søges automatisk efter "%IDENT%". Der skælnes
+    ikke mellem små og store bogstaver.
+
+    Antallet af søgeresultater begrænses som standard til 20.
+    """
+    if '%' not in ident:
+        ident = f"%{ident}%"
+
+    ident_pattern = ident.replace("%", ".*")
+
+    punkter = firedb.soeg_punkter(ident, antal)
```

Comment:
I `fire info punkt` advarede jeg om at der var flere punkter tilgængelige, så man kunne genkøre med højere n. Jeg kan ikke rigtigt blive eng med mig selv om hvorvidt det er en påkrævet funktionalitet. Ellers kunne den detekteres ved at sige `firedb.soeg_punkter(ident, antal + 1)` og så gø hvis len(punkter) > antal.

Men hæææ huuu... jeg er sikker på at vi nok skal komme på glatis en dag man har glemt den implicitte værdi på 20, på den anden side er jeg også fortaler for ikke at skrive unødigt ævl ud på skærmen når man kan nøjes med at komme med et resultat.

Så alt i alt er din løsning med at begrænse søgningen henne på databasemetallet nok at foretrække

Reply:
Man kunne jo godt trække ud af databasen hvor mange punkter der matcher søgeordet og vise det til brugeren. Det gør koden lidt mere bøvlet fordi man er nødt til at gentage hele kaldet der gemmer sig bag `FireDb.soeg_punkter()`. Lad os lige se om det i praksis viser at være en mangel eller ej.